Makefile

```diff
@@ -6,3 +6,6 @@ flash:
 
 monitor:
 	serial-monitor -b 115200 -p $(serial-port)
+
+
+# dfu-util -D stm32-test.bin -d "0483:df11" -a 0 -s 0x08000000
```

Comment:
remove?

Reply:
Ah sorry, left that as a note for me to work that into the `Makefile` later. The STM32F4 dev board I have supports USB bootloading via USB DFU, so we can use this command while plugged directly into the board, instead of a USB-serial adapter.